### PR TITLE
Prepare release 16.2.1

### PR DESCRIPTION
**Pull request by khmMouna**

### CODE REVIEW

**Review by rlepinski — APPROVED** [review]

**Review by rlepinski — CHANGES_REQUESTED** [review]
Looks like we are tracking some xcframeworks

File: Airship.Android.Automation.nuspec [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
       <dependencies>
          <group targetFramework="MonoAndroid4.1">
-            <dependency id="airship.android.core" version="16.7.1"/>
-            <dependency id="airship.android.layout" version="16.7.1"/>
+            <dependency id="airship.android.core" version="16.7.2"/>
+            <dependency id="airship.android.layout" version="16.7.2"/>
             <dependency id="Xamarin.AndroidX.CustomView" version="1.1.0.4" />
             <dependency id="Xamarin.AndroidX.Room.Runtime" version="2.3.0.4"/>
```
Comment (rlepinski) [inline Airship.Android.Automation.nuspec:17]: need to bump room here

File: Airship.Android.Chat.nuspec [hunk 2]
```diff
@@ -11,14 +11,14 @@
...
-            <dependency id="airship.android.core" version="16.7.1"/>
+            <dependency id="airship.android.core" version="16.7.2"/>
             <dependency id="Xamarin.Kotlin.StdLib" version="1.5.31.2" />
             <dependency id="Xamarin.Kotlin.StdLib.Jdk8" version="1.5.31.2" />
             <dependency id="Xamarin.KotlinX.Coroutines.Core" version="1.5.2.2" />
             <dependency id="Xamarin.KotlinX.Coroutines.Android" version="1.5.2.2" />
             <dependency id="Xamarin.AndroidX.ConstraintLayout" version="2.1.1.2" />
             <dependency id="Xamarin.AndroidX.Paging.Runtime" version="2.1.2.7" />
-            <dependency id="Xamarin.AndroidX.Room.Runtime" version="2.3.0.4"/>
+            <dependency id="Xamarin.AndroidX.Room.Runtime" version="2.4.1"/>
```
Comment (jyaganeh) [inline Airship.Android.Chat.nuspec:21]: I think this should be `2.4.3` to match the Room version used in Chat

**Review by jyaganeh — APPROVED** [review]
one minor comment, but this lgtm otherwise 👍

File: Airship.Android.Chat.nuspec (hunk 2 above)
Comment (khmMouna) [reply Airship.Android.Chat.nuspec:21]: Yes 👍

**Review by rlepinski — APPROVED** [review]

**Pull request merged**